Dockerfile

```diff
@@ -31,6 +31,6 @@ COPY ./.docker/nginx.conf /etc/nginx/nginx.conf
 
 COPY --from=build-app /app/dist/browser /usr/share/nginx/html
 
-ENV PORT 8080
-EXPOSE 8080
-CMD sh -c "rm -f /var/log/nginx/* && envsubst '\$PORT' < /etc/nginx/conf.d/configfile.template > /etc/nginx/conf.d/default.conf && nginx -g 'daemon off;'"
+ENV PORT=8080
+EXPOSE $PORT
+CMD ["sh", "-c", "rm -f /var/log/nginx/* && envsubst '\\$PORT' < /etc/nginx/conf.d/configfile.template > /etc/nginx/conf.d/default.conf && nginx -g 'daemon off;'"]
```

Comment:
Because the new form is the [exec-form](https://docs.docker.com/reference/dockerfile/#shell-and-exec-form) rather than the shell form. This is a JSON syntax, so we [must escape backslashes](https://docs.docker.com/reference/dockerfile/#backslashes).

The backslash itself is required to prevent immediate expansion of `$PORT`, because we want to replace the literal variable in the file.